framework/core/instance.cpp

```diff
@@ -248,6 +248,14 @@ Instance::Instance(const std::string &              application_name,
 
 	volkLoadInstance(handle);
 
+#if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)
+	result = vkCreateDebugReportCallbackEXT(handle, &debug_report_create_info, nullptr, &debug_report_callback);
+	if (result != VK_SUCCESS)
+	{
+		throw VulkanException(result, "Could not create Vulkan instance");
```

Comment:
Can you change the error message to hint at the debug report callback creation failure?

Reply:
Looks good after this change :)

Reply:
fixed.